browser-sign-in/samples.config.js

```diff
@@ -9,13 +9,13 @@
  *
  * See the License for the specific language governing permissions and limitations under the License.
  */
- 
+
 export default {
   oidc: {
     clientId: '{clientId}',
     redirectUri: '{redirectUri}',
     endSessionRedirectUri: '{endSessionRedirectUri}',
-    discoveryUri: 'https://{youOktaOrg}.okta.com/',
+    discoveryUri: 'https://{youOktaDomain}/oauth2/default',
```

Comment:
We should change this to 
https://{yourOktaDoman}/oauth2/default/.well-known/openid-configuration
This way iOS and android will use the same configuration.

Reply:
hmm, why not construct discovery url (append `.well-known/openid-configuration`) in the code instead of config.

Reply:
I think ios doesn't construct the discoveryUri. @IldarAbdullin-okta 
So for react-native. This is to make the discoveryUri the same for both platforms

Reply:
discoveryUri param from RN is mapped to issuer param in iOS. `issuer` param should be in one of the following formats:
`https://{youOktaOrg}.okta.com` or `https://{youOktaDomain}/oauth2/default`
iOS constructs `https://{yourOktaDoman}/oauth2/default/.well-known/openid-configuration` during actual Oauth session

Reply:
Ok great. This is same for android. `https://{yourOktaDoman}/oauth2/default` will append `.well-known/openid-configuration`